assets/css/objects/_form.scss

```diff
@@ -0,0 +1,15 @@
+.form {
+  h2 {
+    margin: 0 auto;
+    max-width: $base-form-width;
+    padding: $base-spacing;
+  }
+
+  form {
+    border: $base-solid-border;
+    border-radius: $base-border-radius;
+    margin: 0 auto;
```

Comment:
```diff
-margin: 0 auto;
+margin-left: auto;
+margin-right: auto;
```

…to prevent [accidental resets](https://css-tricks.com/accidental-css-resets/) and be more explicit about what styles we are trying to achieve?